content/docs/pulumi-cloud/get-started/onboarding-guide.md

```diff
@@ -0,0 +1,326 @@
+---
+title: Setting Up Pulumi for Your Organization
+meta_desc: Learn how to onboard your entire team to Pulumi, and build out your infrastructure platform, with built-in security, best practices, and compliance.
+menu:
+    cloud:
+        name: Onboarding guide
+        parent: pulumi-cloud-get-started
+        weight: 1
+        identifier: pulumi-onboarding-guide
+---
+
+In this guide, you’ll find everything you need to know about using Pulumi within your organization, from setting things up to recommended usage patterns and practices.
+
+## In this guide
+
+* Setup and start using Pulumi Cloud, whether SaaS or self-hosted
+* Set and achieve onboarding goals for your team’s first 30 days to three months
+* Understand and adopt recommended practices for Pulumi’s suite of products
+* Work through (and prepare for) common first-time user challenges
+
+## Pro tips
```

Comment:
Might drop this into a notes tip block:

{{% notes type="tip" %}}
**Pro tips:**
To help everyone quickly adopt Pulumi in your organization, identify a group of “Pulumi champions” across your organization who will encourage other team members to get started right away. The faster your team uses Pulumi, the faster you can ship.
Check out the Pulumi Glossary for specific terms we use across our sites and documentation.
{{% /notes %}}